index.html

```diff
@@ -1072,10 +1080,17 @@ <h2><dfn>Predicted events</dfn></h2>
             <p>The number of events in the list and how far they are from the current timestamp are determined by
             the user agent and the prediction algorithm it uses.</p>
 
-            <p>The events in the predicted events list of a trusted event will have monotonically increasing {{Event/timeStamp}}s [[DOM]],
-            so the first event will have the smallest <code>timeStamp</code>. All predicted events have a <code>timeStamp</code>
-            that is greater than the <code>timeStamp</code> of the dispatched pointer event that the
-            <code><a data-lt="PointerEvent.getPredictedEvents">getPredictedEvents</a></code> method was called on.</p>
+            <p>The events in the predicted events list of a trusted event will have:</p>
+            <ul>
+                <li>Monotonically increasing {{Event/timeStamp}} values [[DOM]] — all predicted events have a
+                    {{Event/timeStamp}} that is greater than the {{Event/timeStamp}} of the dispatched pointer event that the
```

Comment:
Less likely to occur than with coalesced events, but I think the constraint is technically not less than?

Reply:
as above... `>=` ... `greater than, or equal to, the...` ?